deploy/helm/sumologic/conf/logs/logs.source.containers.conf

```diff
@@ -34,6 +34,18 @@
     </regexp>
   </filter>
 {{- end }}
+
+{{ if .Values.otelcol.logLevelFilter }}
+ {{ printf "<filter **%s**>" (include "sumologic.metadata.name.otelcol.deployment" .) }}
```

Comment:
Shouldn't it be just `sumologic.metadata.name.otelcol`?

Reply:
should be, also I need to add it to the _helpers

Reply:
It doesn't matter too much right now, because there is only one otelcol, also `sumologic.metadata.name.otelcol.deployment` is already in _helpers

Reply:
Should we change it as @perk-sumo suggests above or leave it as it is?

Reply:
Let me add commits and merge.